rusk-wallet/src/main.rs

```diff
@@ -72,23 +66,23 @@ pub(crate) struct WalletCfg {
     prover_port: Option<String>,
 
     /// IPC method for communication with rusk [uds, tcp_ip]
-    #[clap(short = 'i', long, default_value = "uds")]
-    ipc_method: String,
+    #[clap(short = 'i', long)]
```

Comment:
Shouldn't we have a default setting that works out of the box?

Reply:
Yes, defaults are handled in the program. We deprecated `clap` defaults because there was no way of telling whether the user had chosen that option or it was just us defaulting to it.